src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -6577,12 +6577,10 @@ public class A : Attribute
 class C<T, U> { public enum E {} }";
 
             CreateCompilation(source).VerifyDiagnostics(
-                // (7,31): error CS0182: An attribute argument must be a constant expression, typeof expression or array creation expression of an attribute parameter type
-                // [A(new object[] { default(E), default(E) })]
-                Diagnostic(ErrorCode.ERR_BadAttributeArgument, "default(E)").WithLocation(7, 31),
                 // (7,19): error CS0182: An attribute argument must be a constant expression, typeof expression or array creation expression of an attribute parameter type
                 // [A(new object[] { default(E), default(E) })]
-                Diagnostic(ErrorCode.ERR_BadAttributeArgument, "default(E)").WithLocation(7, 19));
+                Diagnostic(ErrorCode.ERR_BadAttributeArgument, "default(E)").WithLocation(7, 19)
+                );
```

Comment:
Please test errors in two separate arguments. This should result in two errors.
```
public A(object[] a, object[] b) { }
...
[A(new object[] { default(E) }, new object[] { default(E) })]
...
```

Reply:
Test added.